src/components/Tool/Appeal/AddAppealForm.tsx

```diff
@@ -452,17 +477,15 @@ const AddAppealForm = (): ReactElement => {
                     multiple
                     autoSelect
```

Comment:
Remove `autoSelect`. My opinion is that autoSelect should be removed from Autocompletes unless they are freeSolo.